### ISSUE TITLE
Yarn build fail with `The "to" argument must be of type string. Received type object`

### ISSUE THREAD

**Issue by trepmag** [apache-superset/superset-ui-plugins#313]
**Describe the bug**
`yarn install && yarn build` fail with the following message:
```
[1/2] CONFIG Creating config files (failed)

The "to" argument must be of type string. Received type object
TypeError [ERR_INVALID_ARG_TYPE]: The "to" argument must be of type string. Received type object
    at validateString (internal/validators.js:112:11)
    at Object.relative (path.js:1055:5)
    at /home/cgampert/Projects/grid/superset/superset-ui-plugins/node_modules/@superset-ui/build-config/configs/typescript.js:24:33
    at Array.forEach (<anonymous>)
    at Object.<anonymous> (/home/cgampert/Projects/grid/superset/superset-ui-plugins/node_modules/@superset-ui/build-config/configs/typescript.js:23:22)
    at Module._compile (internal/modules/cjs/loader.js:945:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:962:10)
    at Module.load (internal/modules/cjs/loader.js:798:32)
    at Function.Module._load (internal/modules/cjs/loader.js:711:12)
    at Module.require (internal/modules/cjs/loader.js:838:19)

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

This was triggered on current master head (df66066) while following [steps](https://github.com/apache-superset/superset-ui-plugins/tree/master/packages/superset-ui-plugins-demo) of the demo package.

Update: the error message is different with a node version 12 (instead of a 8 previously)

**Comment by kalimuthu123** [comment]
![beemo](https://user-images.githubusercontent.com/42966797/73650864-2c027f80-46a9-11ea-9768-a4cb8c83a996.png)
i am having this error Beemo.module cannot found

**Comment by trepmag** [comment]
Looks like build fix is being threaten in this issue #319; checkout the `kristw--nimbus` branch...
